dashboard/app/helpers/users_helper.rb

```diff
@@ -42,6 +49,7 @@ def summarize_user_progress_for_all_scripts(user)
   private def merge_user_summary(user_data, user)
     if user
       user_data[:disableSocialShare] = true if user.under_13?
+      user_data[:lockableAuthorized] = user.authorized_teacher? || user.student_of_authorized_teacher?
```

Comment:
Worth making `user.lockable_authorized?` a method?  (If this will be re-used anywhere else.)